docs/development_suite/overview-dev-suite.md

```diff
@@ -4,28 +4,47 @@ title: Console Overview
 sidebar_label: Console Overview
 sidebar_order: 0
 ---
-Welcome to the Mia-Platform Console Overview page!
+This page will give you a brief overview of **Mia-Platform Console**, explain **why it's important**, highlight the **advantages of adopting this internal platform solution** and provide a **brief description of its architecture**. 
 
-![Mia-Platform Console Overview](img/Mia-Platform-console-overview.png)
+## Product overview 
 
-In an omnichannel and evolutionary market, successful enterprises are the ones with the capabilities to operate and innovate like tech companies:
+In an evolving, omnichannel market, successful enterprises must operate and innovate like tech companies by:
 
-* **Speed**: Being able to build and launch new digital products with great speed;
-* **Evolvability**: Continuously looking at the ongoing market changes as opportunities;
-* **Scalability**: Being flexible and ready to answer rapidly to demand expansions or contractions;
-* **Openness**: Being able to integrate with tools, partner channels, platforms and ecosystems.
+* Building and launching digital products **quickly**.
+* Viewing market changes as **opportunities**.
+* Being **flexible** to rapidly respond to demand fluctuations.
```

Comment:
```suggestion
* Being **flexible** to rapidly respond to demand fluctuations, easily increasing or decreasing system capacity based on traffic fluctuations.
```

Reply:
Ok, what about this version?
* Having the **flexibility** to quickly adjust to changes in demand by easily increasing or decreasing scaling system capacity in response to traffic fluctuations.